src/cowboy_rest.erl

```diff
@@ -328,7 +328,9 @@ uri_too_long(Req, State) ->
 allowed_methods(Req, State=#state{method=Method}) ->
 	case call(Req, State, allowed_methods) of
 		no_call when Method =:= <<"HEAD">>; Method =:= <<"GET">> ->
-			next(Req, State, fun malformed_request/2);
+			Allow = <<"HEAD, GET, OPTIONS">>,
```

Comment:
Am I right to assume this? If the `allowed_methods` was not provided, the default headers will be HEAD, GET, and OPTIONS.

cc @essen